examples/ads.amp.html

```diff
@@ -835,8 +835,9 @@ <h2>BeaverAds</h2>
       data-block-id="1001038">
   </amp-ad>
 
+  <h2>BeOp</h2>
   <amp-ad width="300" height="1"
-      type="beopinion"
+      type="beop"
```

Comment:
I don't see anything being rendered here. Is it possible to return an example ad? It helps us amp side to catch regressions etc.